### PR TITLE
Return after catching UpgradeRequiredError

### PR DESCRIPTION
**Pull request by vitoreiji**
Fix re-throwing UpgradeRequiredError when adding an alias from the custom domain wizard.

tutadb#1569

### CODE REVIEW

File: src/settings/emaildomain/AddEmailAddressesPage.ts [hunk 1]
```diff
@@ -184,6 +184,7 @@ export class AddEmailAddressesPageAttrs implements WizardPageAttrs<AddDomainData
 					return false
 				} else if (e instanceof UpgradeRequiredError) {
 					await showPlanUpgradeRequiredDialog(e.plans, e.message)
+					return false
```
Comment (charlag) [inline src/settings/emaildomain/AddEmailAddressesPage.ts:187]: that's what you get for not having an `else` case huh

Comment (vitoreiji) [reply src/settings/emaildomain/AddEmailAddressesPage.ts:187]: I mean, the else case would be throwing the error, so...

Comment (charlag) [reply src/settings/emaildomain/AddEmailAddressesPage.ts:187]: yeah but you wouldn't just fallthrough to it

**Review by vitoreiji — COMMENTED** [review]
Is this better, then, @charlag?

**Comment by charlag** [comment]
I would  still do  everything in each brunch but maybe this is better?

**Comment by paw-hub** [comment]
Merged via https://github.com/tutao/tutanota/pull/5794

**Pull request closed without merging**